src/pseudopeople/noise_functions.py

```diff
@@ -116,22 +116,33 @@ def generate_incorrect_selections(
 #     return column
 
 
-# def swap_months_and_days(
-#     column: pd.Series,
-#     configuration: ConfigTree,
-#     randomness_stream: RandomnessStream,
-#     additional_key: Any,
-# ) -> pd.Series:
-#     """
+def swap_months_and_days(
+    column: pd.Series,
+    configuration: ConfigTree,
+    randomness_stream: RandomnessStream,
+    additional_key: Any,
+) -> pd.Series:
+    """
+    Function that swaps month and day of dates.
 
-#     :param column:
-#     :param configuration:
-#     :param randomness_stream:
-#     :param additional_key: Key for RandomnessStream
-#     :return:
-#     """
-#     # todo actually duplicate rows
-#     return column
+    :param column: pd.Series containing dates with the format YYYY-MM-DD
+    :param configuration: ConfigTree object containing noise level values
+    :param randomness_stream: Randomness Stream object for random choices using vivarium CRN framework
+    :param additional_key: Key for RandomnessStream
+    :return: Noised pd.Series where some dates have month and day swapped.
+    """
+    if column.name == "event_date":
```

Comment:
This if statement should use the date format to determine where month and day is, not the name of the column. What if we introduced a new dataset that had a column called `event_date` which had its date in `MM/DD/YYYY` format?